tutorials/plot_epoching_and_averaging.py

```diff
@@ -41,7 +41,15 @@
 # attention in aligning the events correctly with the raw data.
 events = mne.find_events(raw)
 print(events)
-mne.viz.plot_events(events, raw.info['sfreq'], raw.first_samp)
+
+# Plot the events to get an idea of the paradigm
+# Specify colors and an event_id dictionary for the legend.
+event_id = {'aud_l': 1, 'aud_r': 2, 'vis_l': 3, 'vis_r': 4, 'smiley': 5,
```

Comment:
Let's tackle this is a subsequent PR since we do this in a lot of places currently